.mocharc.js

```diff
@@ -1,23 +0,0 @@
-'use strict';
-
-module.exports = {
-  require: [
-    'ts-node/register',
-    'source-map-support/register',
-  ],
-  extension: [
-    'ts'
-  ],
-
-  // Do not look for mocha opts file
-  opts: false,
-
-  // Warn if test exceed 75ms duration
-  slow: 75,
-
-  // Fail if tests exceed 10000ms
-  timeout: 10000,
-
-  // Check for global variable leaks
-  'check-leaks': true,
-}
```

Comment:
Why was this deleted?

Reply:
hmm, I also don't know.  Also something I didn't touch.